R/colformat.R

```diff
@@ -0,0 +1,44 @@
+cf_data.hms <- function(x, ...) {
```

Comment:
I think you need a comment here along the lines of "dynamically exported; see `zzz.R`

Reply:
Done.